ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -243,7 +243,9 @@
     - cloud_platform_is_aws
     - not (use_sbd | bool)
   register: stonith_config_result
-  failed_when: "'ERROR' in stonith_config_result.stderr"
+  failed_when: >
+    (ansible_distribution_major_version == '12' and 'ERROR' in stonith_config_result.stderr and 'ERROR: warning' not in stonith_config_result.stderr) or
+    (ansible_distribution_major_version != '12' and 'ERROR' in stonith_config_result.stderr)
```

Comment:
This part addresses **Problem 1** (see description): it specifically checks if 12sp5 `crm` outputs a warning in stderr